SLHCUpgradeSimulations/Configuration/python/phase2TkFlat.py

```diff
@@ -1,4 +1,5 @@
 import FWCore.ParameterSet.Config as cms
+from Configuration.StandardSequences.Eras import eras
```

Comment:
It is not used now but I would like to keep it because every time that I I move part of the custom file, I use 

    if not eras.trackingPhase2PU140.isChosen():

to comment out the parts that I move.